osbs/api.py

```diff
@@ -214,28 +218,30 @@ def _set_build_request_resource_limits(self, build_request):
                                               storage=storage_limit)
 
     @osbsapi
-    def get_build_request(self, build_type=None, inner_template=None,
-                          outer_template=None, customize_conf=None,
-                          arrangement_version=DEFAULT_ARRANGEMENT_VERSION):
+    def get_build_request(
+            self, build_type=None, inner_template=None,
+            outer_template=None,
+            arrangement_version=DEFAULT_ARRANGEMENT_VERSION,
+            user_params=None,
```

Comment:
This is API breaking change, are we fine with that? (/me is ok).

I don;t think that `user_params` can be None here. I haven't found code that will handle `None`